internal/uvm/constants.go

```diff
@@ -34,4 +36,43 @@ const (
 var (
 	errNotSupported = errors.New("not supported")
 	errBadUVMOpts   = errors.New("UVM options incorrect")
+
+	// V5 GUIDs for SCSI controllers
+	// These GUIDs are created with namespace GUID "d422512d-2bf2-4752-809d-7b82b5fcb1b4"
+	// and index as names. For example, first GUID is created like this:
+	// guid.NewV5("d422512d-2bf2-4752-809d-7b82b5fcb1b4", []byte("0"))
+
```

Comment:
remove new line to fix docstring?

Reply:
or I guess you might get rid of this if you're working on function that does `guid.NewV5(...)` call to return controller GUID.

Reply:
I have removed the GUIDs and now just maintain them as strings since we didn't really need the actual GUID structs anyway.